### ISSUE TITLE
Feature suggestion/question: access to initialize/addTranslationForLanguage without HOC

### ISSUE THREAD

**Issue by benkeen** [ryandrewjohnson/react-localize-redux#96]
Hi, great project! Quick feature request / "can-I-do-this" question. 

We were using an earlier version of this lib which allowed for initializing the global localization data like so:

```javascript
import { initialize, addTranslationForLanguage } from 'react-localize-redux';
... 
store.dispatch(initialize(...));
store.dispatch(addTranslationForLanguage(en, 'en'));
```

This was exceedingly convenient. But with the latest version, I see it now requires a HOC using `withLocalize` to initialize and set the data:
https://ryandrewjohnson.github.io/react-localize-redux-docs/#initialize-localize

The reason we favour the previous method is that we're integrating React into an existing system, piece-meal, and won't have any form of a top-level component where such initialization would take place. We _could_ hack it, and create a standalone component whose only purpose is to initialize the localization, but it would mean extra legwork in getting the component "rendered" in our pipeline. Our preference would be the former: having those methods exposed like before so when after we set up our store, just initialize our localization content - all independent of React. 

Any suggestions?

**Comment by ryandrewjohnson** [comment]
Thanks for the question.

So now that the latest version is no longer dependent on Redux the entire React implementation is all taken care of in `LocalizeProvider`. This was to keep the API, and instructions for use as similar as possible for both Redux and non-Redux users.

That being said there are a couple options if the `withLocalize` HOC doesn't work for you:


1. You can access use [LocalizeContext.Consumer](https://ryandrewjohnson.github.io/react-localize-redux-docs/#localizecontext) directly in your component. From there you will have access to context which has access to `initilaize`. (**recommended**)

2. Although no longer documented the original [action creators are exported](https://github.com/ryandrewjohnson/react-localize-redux/blob/master/src/index.js). You could try to import these directly, and dispatch them from your store. 

The reason I recommend option 1 is that it will still work regardless of whether you're still using Redux. Also I haven't tested option 2, so not sure if it will work. If you do end up going with option 2 let me know if it works.

**Comment by benkeen** [comment]
Thanks, @ryandrewjohnson! I appreciate the explanation, that make sense.

It doesn't look like option 2 will work: after upgrading I noticed the actions were firing fine, but the store didn't end up getting the expected `locale` top-level property. It's possible I goofed somewhere, but if it's an undocumented option, I'd have shied away from using it anyway. So it's probably good. :)

I think with option 1, we'll still need some sort of component in which to access the context. That's kind of what we're hoping to do away with, but no worries. I'll chat with the peeps and think things over. Thanks!
